programs/mango-v4/src/serum3_cpi.rs

```diff
@@ -155,6 +156,22 @@ impl OpenOrdersSlim {
             referrer_rebates_accrued: oo.referrer_rebates_accrued,
         }
     }
+    pub fn from_oo_v2(oo: &OpenOrdersV2, base_lot_size: i64, quote_lot_size: i64) -> Self {
+        let base_locked_native: u64 = (oo.position.bids_base_lots * base_lot_size)
```

Comment:
unfortunate that these are i64

Reply:
I changed it to use u64, think it's a bit nicer